### PR TITLE
feat: better print `traituse` node

### PR DESCRIPTION
**Pull request by evilebottnawi**
:star:

### CODE REVIEW

File: tests/traits/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -86,7 +86,9 @@ class ImplementingClass
 
 class ShortImplementingClass
 {
-    use testTrait { A::testFunction insteadof C; }
+    use testTrait {
+        A::testFunction insteadof C;
+    }
```
Comment (evilebottnawi) [inline tests/traits/__snapshots__/jsfmt.spec.js.snap]: We always add `newline` after `{` (in function, class, closure and etc), let's do this here for consistently output

Comment (evilebottnawi) [reply tests/traits/__snapshots__/jsfmt.spec.js.snap]: I am wrong here, better use 
```
use testTrait {A::testFunction insteadof C;}
```

Comment (evilebottnawi) [reply tests/traits/__snapshots__/jsfmt.spec.js.snap]: From PSR-12:
> When using the insteadof and as operators they must be used as follows taking note of indentation, spacing and new lines.

https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#42-using-traits

Comment (czosel) [reply tests/traits/__snapshots__/jsfmt.spec.js.snap]: :+1: Which is eaxtly what you implemented now, right?

Comment (evilebottnawi) [reply tests/traits/__snapshots__/jsfmt.spec.js.snap]: @czosel Yep :+1:

**Pull request merged**